man/systemd-firstboot.xml

```diff
@@ -264,6 +264,14 @@
         <literal>root</literal> user instead of overwriting the entire file.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--reset</option></term>
+
+        <listitem><para>If specified, all existing files that are configured by
+        <command>systemd-firstboot</command> are removed. Note that the files are removed regardless of
+        whether they'll be configured with a new value or not.</para></listitem>
```

Comment:
maybe say: this ensures that the next boot-up of the image will be considered a first boot, and systemd-firstboot will ask its questions again. or something like that.